packages/fe-fpm-writer/test/unit/subsection.test.ts

```diff
@@ -0,0 +1,345 @@
+import os from 'os';
```

Comment:
That looks pretty similar to the section tests. Do we really need them or would it be sufficient just to test the additional selection of the template if it is a subsection because otherwise, it is the same coding, correct?